contrib/NeonVM/neonvm.c

```diff
@@ -1,113 +1,125 @@
+#include <stdarg.h>
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
 
 #include "opcode.h"
 #include "stack.h"
+#include "util.h"
 
-#define error(s) fprintf(stderr, "%s\n", s); abort();
+typedef unsigned char       uint8_t;
+typedef signed   char       int8_t;
+typedef unsigned short      uint16_t;
+typedef signed   short      int16_t;
+typedef unsigned int        uint32_t;
+typedef signed   int        int32_t;
+typedef unsigned long long  uint64_t;
+typedef signed   long long  int64_t;

```

Comment:
From Travis:

contrib/NeonVM/neonvm.c:17:29: error: conflicting types for 'int64_t'
 typedef signed   long long  int64_t;